client/my-sites/theme/main.jsx

```diff
@@ -312,8 +313,12 @@ const ThemeSheet = React.createClass( {
 		const priceElement = <span className="themes__sheet-action-bar-cost">{ this.props.price }</span>;
 		const siteID = this.props.selectedSite && this.props.selectedSite.ID;
 
+		const analyticsPath = `/theme/:slug${ section ? '/:section' : '' }${ siteID ? '/:site_id' : '' }`;
+		const analyticsPageTitle = `Themes > Details Sheet${ section ? ' > Section' : '' }${ siteID ? ' > Site' : '' }`;
```

Comment:
Yes. :P 

(Skypping "Overview" since it's just the root "Sheet": `Themes > Sheet > $siteid`)